library/src/scala/compiletime/package.scala

```diff
@@ -50,7 +55,9 @@ package object compiletime {
     * @note only `inline` arguments will be displayed as "code".
     *       Other values may display unintutively.
     */
-    transparent inline def code (inline args: Any*): String = ???
+    transparent inline def code (inline args: Any*): String =
+      // implemented in dotty.tools.dotc.typer.Inliner.Intrinsics
+      error("`code` was not evaluated by the compiler")
```

Comment:
You could add a prefix `Compiler bug: ` to these messages to make it clear it's not the user fault.

Reply:
Added